Storage/Storage/Model/WooCommerce.xcdatamodeld/Model 20.xcdatamodel/contents

```diff
@@ -203,7 +203,7 @@
         <attribute name="backordersKey" attributeType="String"/>
         <attribute name="briefDescription" optional="YES" attributeType="String"/>
         <attribute name="catalogVisibilityKey" attributeType="String"/>
-        <attribute name="crossSellIDs" optional="YES" attributeType="Transformable" customClassName="[Int64]"/>
+        <attribute name="crossSellIDs" optional="YES" attributeType="Transformable" valueTransformerName="NSSecureUnarchiveFromData" customClassName="[Int64]"/>
```

Comment:
great catch on the unit tests and thanks a lot for the example! I didn't think of adding one since the Core Data warnings were in the console, but it's a good idea to ensure the transformable attributes stay the same after migration throughout the affected model versions.

since I added similar setup for populating data in [a previous PR](https://github.com/woocommerce/woocommerce-ios/pull/2474), I added a test case on the transformable attributes to `CoreDataIterativeMigratorTests` in: https://github.com/woocommerce/woocommerce-ios/pull/2475/commits/74a72233dfbcba4f4f05d2c8f7c0f246394fde4e. Lemme know if you think separating the test into a new test class or split into two test cases for `Product` and `ProductAttribute` would be better. The test case initializes the database to model version 20 (the earliest affected model version), populates the required and transformable attributes for `Product` and `ProductAttribute`, then goes through a migration from model version 20 to 28 (the latest affected model version). If you have any suggestions, I'm happy to update the test!

Reply:
> Lemme know if you think separating the test into a new test class or split into two test cases for Product and ProductAttribute would be better.

The one I showed and the one you created are two different things. My gist was focused on persisting while the one you made is about the migration. That said, I think it's fine to just have the migration so we don't have too many tests that are potentially slow. 🙂 

Thanks for adding that!

Reply:
yea I diverged from the gist you shared, the test case now saves the context after inserting a `Product` and `ProductAttribute` with model. version 20, then performs a migration to model version 28 to assert the transformable attributes. I think it'd be nice to separate the required Core Data entity fields setup to a utility class the next time we add more tests to this.